ui/react-components/components/Provider/ProviderSearch.jsx

```diff
@@ -1,73 +1,133 @@
-import {getAllProviders} from "../../api/providerApi";
+// import { getAllProviders } from "../../api/providerApi";
+import { getAllProviders } from "../../api/__mocks__/providerApi";
 import Dropdown from "../Dropdown/Dropdown.jsx";
-import React, {useEffect, useState} from "react";
+import React, { useEffect, useState } from "react";
 import Tags from "../Tags/Tags.jsx";
-import {forEach, find} from 'lodash';
+import { forEach, find } from "lodash";
 import PropTypes from "prop-types";
-import {injectIntl} from "react-intl";
-import {sortBy} from "lodash";
-import {getValidProviders} from "../../helper";
-import {PROVIDER_RESPONSES} from "../../constants";
+import { injectIntl } from "react-intl";
+import { sortBy } from "lodash";
+import { getValidProviders } from "../../helper";
+import { PROVIDER_RESPONSES } from "../../constants";
+import {
+  searchFeildOnChangeHandler,
+  searchFeildOnRemoveHandler
+} from "../../helper";
 
 const ProviderSearch = props => {
+  const {
+    intl,
+    selectedProviders,
+    onChange,
+    onProviderRemove = e => onRemoveHandler(e),
+    isDisabled
+  } = props;
+  const {
+    openMenuOnClick = true,
+    openMenuOnFocus = true,
+    style = "",
+    components,
+    customSelectStyle
+  } = props;
 
-    const {intl, selectedProviders, onChange, onProviderRemove, isDisabled} = props;
-    const placeHolder = intl.formatMessage({
-        id: 'PLACEHOLDER_APPOINTMENT_CREATE_SEARCH_PROVIDER', defaultMessage: 'Choose Provider'
+  const placeHolder = intl.formatMessage({
+    id: "PLACEHOLDER_APPOINTMENT_CREATE_SEARCH_PROVIDER",
+    defaultMessage: "Choose Provider"
+  });
+
+  const createDropdownOptions = results => {
+    const options = [];
+    forEach(results, provider => {
+      if (
+        provider.attributes.length > 0
+          ? provider.attributes[0].attributeType.display ===
+            "Available for appointments"
```

Comment:
There is an existing constant present for this [here](https://github.com/Bahmni/openmrs-module-appointments-frontend/blob/ed15e4486c5011f9f3720fb5de53f6ff41af8d68/src/constants.js#L37). It would be better to use that instead of duplicating it.

Reply:
yes. Modified

Reply:
Thanks!